Update README.md add comment for VS clone option (#584)
AgOpenGPS-Official/AgOpenGPS: README.md

[![GitHub Release](https://img.shields.io/github/v/release/agopengps-official/AgOpenGPS)](https://github.com/agopengps-official/AgOpenGPS/releases/latest)

> # USE VISUAL STUDIO TO CLONE THIS REPOSITORY!!

Author: Yes but for the time being it’s good to attent people to do it this way. And people don’t read text.. :)

> # USE VISUAL STUDIO TO CLONE THIS REPOSITORY!!

Reviewer: But if they don't read text, why assume that they read it just because it is all caps?

This is kind of like adding some lines to tell people: "If you encounter this bug, do that to resolve it" at the top of the README. You wouldn't do that either, would you?

Even on your desktop
3. Run AgOpenGPS.exe

## Add your own code

> 1. You need to use Visua Studio to clone this repository.

Reviewer: You don't technically "need" to use Visual Studio to clone the repository. There are many other ways of doing the clone. Using Visual Studio is just the simplest for most people.
Maybe we could write it like this:
```suggestion
1. Clone this repository (e.g. use Visual Studio to do so)
```
